Makefile

```diff
@@ -6,7 +6,7 @@ KERNEL_ABI_MINOR_VERSION = 2
 KVERSION_SHORT ?= 5.10.0-23-$(KERNEL_ABI_MINOR_VERSION)
 KVERSION ?= $(KVERSION_SHORT)-amd64
 KERNEL_VERSION ?= 5.10.179
-KERNEL_SUBVERSION ?= 1
```

Comment:
What is the subversion, and why change it?

Reply:
The subversion refers to the Debian version number. In this case, version 5.10.179-3 from Debian is being used.